version/constants.go

```diff
@@ -5,9 +5,9 @@ const (
 	RosettaVersion = "v1.4.12"
 
 	// RosettaMiddlewareVersion is the version of this package (application)
-	RosettaMiddlewareVersion = "v0.2.5"
+	RosettaMiddlewareVersion = "v0.2.6"
 
 	// NodeVersion is the canonical version of the node runtime
 	// TODO: We should fetch this from node/status.
-	NodeVersion = "rc/2022-july"
+	NodeVersion = "v1.3.42"
```

Comment:
if you set this to v1.3.42 please update go.mod to fetch the elrond-go v1.3.42 for consistency reasons

Reply:
`NodeVersion` is the version of the Observer to which rosetta connects, not the referenced version of elrond-go (used as a library).

However, the update in go.mod will be done as soon as possible, in a separate PR.

:pray: